izpack-core/src/test/java/com/izforge/izpack/core/data/DefaultVariablesTest.java

```diff
@@ -267,8 +267,8 @@ public void testDynamicVariablesIZPACK1260()
         rules.readConditionMap(conditions);
         ((DefaultVariables) variables).setRules(rules);
 
-        variables.add(createDynamicCheckonce("previous.wrapper.conf", "${INSTALL_PATH}/conf/wrapper.conf".replaceAll("/+", File.separator), "haveInstallPath"));
-        variables.add(createDynamicCheckonce("previous.wrapper.conf", "${INSTALL_PATH}/wrapper.conf".replaceAll("/+", File.separator), "haveInstallPath+!previous.wrapper.conf.exists"));
+        variables.add(createDynamicCheckonce("previous.wrapper.conf", "${INSTALL_PATH}/conf/wrapper.conf".replace("/", File.separator), "haveInstallPath"));
+        variables.add(createDynamicCheckonce("previous.wrapper.conf", "${INSTALL_PATH}/wrapper.conf".replace("/", File.separator), "haveInstallPath+!previous.wrapper.conf.exists"));
```

Comment:
java.lang.StringIndexOutOfBoundsException: String index out of range: 1
	at java.lang.String.charAt(String.java:686)
	at java.util.regex.Matcher.appendReplacement(Matcher.java:703)
	at java.util.regex.Matcher.replaceAll(Matcher.java:813)
	at java.lang.String.replaceAll(String.java:2189)
	at com.izforge.izpack.core.data.DefaultVariablesTest.testDynamicVariablesIZPACK1260(DefaultVariablesTest.java:270)